### PR TITLE
update to @theia 1.13.0

### PR DESCRIPTION
**Pull request by marcdumais-work**
#### What it does

Update to latest @theia,  v1.13.0

#### How to test

Build this PR and confirm that it's still possible to successfully package and run packaged version of `Blueprint`. 

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Review by thegecko — APPROVED** [review]

**Comment by marcdumais-work** [comment]
Jenkins CI is not passing ATM. Problem when rebuilding native dependencies. i'm trying to update dependencies further than @theia packages, in the hope it may help. Pushing an update soon.

**Comment by marcdumais-work** [comment]
@thegecko I see mentions of cache in the Jenkins file. Is there a way to flush that cache? I think it may be in the way for this PR.

**Comment by thegecko** [comment]
> @thegecko I see mentions of cache in the Jenkins file. Is there a way to flush that cache? I think it may be in the way for this PR.

@marcdumais-work I'm not sure what you mean. The cache mounts in the linux pod are needed for the tools to work and (I believe) are not persisted.

The only other cache command is the yarn one which explicitly clears the yarn cache.

**Comment by thegecko** [comment]
I see the error is always with linux and it failed to build `node-pty`, I don't see mention of a cache, but this *might* be the cause?

TBF, I hate that the linux build has to be done in this constrained pod, it's not something I have much knowledge around.

**Comment by marcdumais-work** [comment]
So, we should be getting a "from scratch" build every time (e.g. from a fresh repo clone)? If so that's good.

**Comment by thegecko** [comment]
> So, we should be getting a "from scratch" build every time (e.g. from a fresh repo clone)? If so that's good.

I believe so, but no guarantees!

**Comment by marcdumais-work** [comment]
Is there more info somewhere about this "constrained pod" we use?

**Comment by thegecko** [comment]
I think it's this:

https://plugins.jenkins.io/kubernetes/

File: Jenkinsfile [hunk 1]
```diff
@@ -135,6 +135,8 @@ spec:
 
 def buildInstaller() {
     checkout scm
+    // clear volumes to avoid occasional CI issue
+    sh "rm -rf /.cache/* /.electron-gyp/*"
```
[the remark this answers is not in the document]
Comment (thegecko) [reply Jenkinsfile:139]: Sure

**Comment by thegecko** [comment]
Nice work!

**Pull request merged**